Kernel/Tasks/Process.h

```diff
@@ -129,6 +129,7 @@ class Process final
         Atomic<u32> thread_count { 0 };
         u8 termination_status { 0 };
         u8 termination_signal { 0 };
+        UnixDateTime creation_time;
```

Comment:
If we put this out of the protected members' block, we can mark this as const as well.